woocommerce/woocommerce-android
# Issue 5817 - [Order Creation] Add content description for edit buttons in order creation screen

Closes: #5817 


### Description

Add content description for edit buttons in order creation screen. Only "Edit" doesn't give much context when using TalkBack. We're adding specific content description strings for each edit button in order creation to give more context for users using TalkBack.

### Testing instructions

First make sure "Order Creation" beta feature is enabled:
1 - Tap "Menu" in the bottom navigation
2 - Tap the settings button
3 - Tap "Beta features" button
4 - Enable "Order Creation"

After "Order Creation" beta feature is enabled:
1 - With TalkBack on, navigate to "Orders" tab
2 - Tap the FloatingActionButton ("+" button)
3 - Open "Create order" option
4 - Highlight the "Edit" button next to the order status. TalkBack should read "Edit order status"
5 - Add a Customer using the "Add customer" button.
6 - Highlight the "Edit" button next to the Customer section. TalkBack should read "Edit customer details"
7 - Add a Customer Note using the "Add note" button
8 - Highlight the "Edit" button next to the Customer Note section. TalkBack should read "Edit customer note"

### Images/gif

N/A
